scripts/setup.sh

```
@@ -358,10 +358,13 @@ EOF
         --dry-run=client -o yaml | oc apply -f -
 
     echo "Deploying OSAC using Helm..."
+    DOMAIN=$(oc get dns/cluster -o jsonpath='{.spec.baseDomain}')
     helm dependency update charts/osac/
     helm upgrade --install osac charts/osac/ \
         --namespace "${INSTALLER_NAMESPACE}" \
         --values "${VALUES_FILE}" \
+        --set service.externalHostname=fulfillment-api-${INSTALLER_NAMESPACE}.${DOMAIN} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)